### ISSUE TITLE
Yolov8n quantized model doesn't work

### ISSUE THREAD

**Issue by hamedgorji** [PINTO0309/onnx2tf#263]
### Issue Type

Others

### onnx2tf version number

1.7.35

### onnx version number

1.13.1

### tensorflow version number

2.11.0

### Download URL for ONNX

https://easyupload.io/qahfq0

### Parameter Replacement JSON

```json
I don't know how I can access this.
```

### Description

I am trying to convert the yolov8n pytorch model to edgetpu. 
First, I use the following command to generate .onnx file `!yolo export model=/content/drive/MyDrive/Yolo/best.pt imgsz=640 format=onnx`
Then I use `!onnx2tf -i '/content/drive/MyDrive/Yolo/best.onnx' -oiqt -o '/content/drive/MyDrive/Yolo/tf_model' ` to generate the quantized model. 
and for the final step, I use ` ! edgetpu_compiler $TFLITE_FILE` to convert it to edgetpu format.
But the problem is one step before converting the model to edgetpu. 
When I use `!onnx2tf -i '/content/drive/MyDrive/Yolo/best.onnx' -oiqt -o '/content/drive/MyDrive/Yolo/tf_model' ` it generates several files, including `best_integer_quant.tflite`, `best_full_integer_quant.tflite`, `best_dynamic_range_quant.tflite`, etc. 
The problem is when I use these files to do the inference (to check if the model works fine), none of them works (the model cannot find the object I want) except `best_dynamic_range_quant.tflite`, which it's inference time is around 3000ms. 
I want to use `best_full_integer_quant.tflite` model with an inference time of around 60ms, but It cannot detect anything. 
I wonder why `best_dynamic_range_quant.tflite` works even if the inference time is high, but the other models don't work.
I've been working on this issue for about two weeks, but I couldn't figure out what can be the possible problem. 
I use several options when I want to convert the onnx file, including `-qt per-tensor --copy_onnx_input_output_names_to_tflite --output_signaturedefs --replace_to_pseudo_operators,` but none of them works. 
I would appreciate it if you could help me to convert the yolov8n model to edgetpu.
PS. I can convert it successfully but it cannot detect anything.

**Comment by PINTO0309** [comment]
Duplicate of https://github.com/PINTO0309/onnx2tf/issues/244

https://github.com/PINTO0309/onnx2tf/tree/1.7.35#6-if-the-accuracy-of-the-int8-quantized-model-degrades-significantly

**Comment by hamedgorji** [comment]
@PINTO0309  Thank you. I will check this solution and let you know if I can solve the problem.
